kolena/dataset/evaluation.py

```diff
@@ -169,8 +169,11 @@ def upload_results(
     :param dataset: The name of the dataset.
     :param model: The name of the model.
     :param results: Either a DataFrame or a list of tuples, where each tuple consists of
-                    an eval configuration and a DataFrame.
-    :return: None
+                    an eval configuration and a DataFrame. The DataFrame type has been aliased to be either a pandas
```

Comment:
Is there a way for `DataFrame` type to be expanded or linkable to the source definition? Exaplaining type alias seems a bit verbose.

Reply:
currently there is a not very descriptive tooltip, seems only the python builtin types gets a link
![Screenshot 2024-01-12 at 11 28 22 AM](https://github.com/kolenaIO/kolena/assets/146848587/99ab1efb-bd6e-4f94-b28a-38070811840b)

Reply:
They are expandable now